XCUITests/NewTabSettings.swift

```diff
@@ -30,6 +30,23 @@ class NewTabSettingsTest: BaseTestCase {
         waitForNoExistence(app.collectionViews.cells["TopSitesCell"].collectionViews.cells["youtube"])
         waitForNoExistence(app.staticTexts["Highlights"])
     }
+    
+    func testChangeNewTabSettingsShowFirefoxHome() {
+        // Set to blank page first since FF Home is default
+        navigator.goto(NewTabSettings)
+        navigator.performAction(Action.SelectNewTabAsBlankPage)
+        navigator.performAction(Action.OpenNewTabFromTabTray)
```

Comment:
The test is failing, since the blank page is open with the cursor in the address bar the tabs button to go to tab tray is not available unless you tap on < button to remove the focus from the address bar or you type something to go to a different website and then open a new tab. I would suggest to change this to use the bookmark or history and then go to Firefox Home